demo/src/store/initialState.js

```diff
@@ -63,15 +63,20 @@ import wheresPoeThumbnail from '../images/thumbnails/wheres_poe.png';
 
 
 /**
- * The items in the store; each has name, image, cost, and
+ * @fileoverview A default initial state for the application.
```

Comment:
Also, this overview seems like it got mixed in with the JSDoc for the ItemStore typedef.

Reply:
Removed the overview